src/main/java/mate/academy/spring/MainApp.java

```diff
@@ -1,56 +1,129 @@
 package mate.academy.spring;
 
 import java.sql.SQLException;
+import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
 
 import mate.academy.spring.config.AppConfig;
+import mate.academy.spring.entity.Author;
 import mate.academy.spring.entity.Book;
+import mate.academy.spring.entity.Rent;
 import mate.academy.spring.entity.User;
-import mate.academy.spring.service.BookService;
-import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.interfaces.AuthorService;
+import mate.academy.spring.service.interfaces.BookService;
+import mate.academy.spring.service.interfaces.RentService;
+import mate.academy.spring.service.interfaces.UserService;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 
 public class MainApp {
+
     public static void main(String[] args) throws SQLException {
+        User bora = new User("Sunil", "Bora",
+                "suni.bora@example.com", new ArrayList<Book>());
+        Book master = new Book("Master and Margarita", new ArrayList<Author>(), 1966, 666.6);
+        Author bulgakov = new Author("Mihail", "Bulgakov", new ArrayList<>());
+        List<Book> booksMaster = new ArrayList<>();
+        booksMaster.add(master);
+        bora.setRentBooks(booksMaster);
+        List<Author> authorsBulgakov = new ArrayList<>();
+        authorsBulgakov.add(bulgakov);
+        master.setAuthors(authorsBulgakov);
```

Comment:
```suggestion
        master.setAuthors(Collections.singletonList(bulgakov));
```
Спробуй чи так спрацює

Reply:
Так, працює.